src/Tizen.NUI/src/public/BaseComponents/Style/Selector.cs

```diff
@@ -169,42 +169,29 @@ public T Other
         [EditorBrowsable(EditorBrowsableState.Never)]
         public T GetValue(ControlState state)
         {
-            if(All != null)
+            if (All != null)
             {
                 return All;
             }
-            switch(state)
+
+            StateValuePair<T> value = Find(x => x.State == state);
+            if (value.Value != null)
+            {
+                return value.Value;
+            }
+
+            if (state.IsCombined)
             {
-                case null:
```

Comment:
looks like this switch-case part has been removed. I couldn't understand the whole logic so I just ask if this is intentionally modified, and if there is replaced logic.

Reply:
I removed switch-case statement intentionally becuase it can be covered by following code,

```C#
// This covers Normal ,Focused, Pressed, Disabled, Selected cases
StateValuePair<T> value = Find(x => x.State == state); 
if (value.Value != null)
{
    return value.Value;
}

// This covers DisabledFocused, DisabledSelected, SelectedFocused cases
if (state.IsCombined)
{
    value = Find(x => state.Contains(x.State));
    if (value.Value != null)
    {
        return value.Value;
    }
}
```

Reply:
OK. I see. Thank you for explanation~